assignments/ev/BUILD

```diff
@@ -0,0 +1,23 @@
+cc_library(
+    name = "euclidean_vector",
+    srcs = ["euclidean_vector.cpp"],
+    hdrs = ["euclidean_vector.h"],
+    deps = [],
+)
+
+cc_binary(
+    name = "client",
+    srcs = ["client.cpp"],
```

Comment:
Maybe rename client to main in order to be consistent with assignment 1?

Reply:
Great point - I had the same thought - but I worry if I call it "main" that they will confuse it with main in the first assignment... which was more focused on it being _the actual program_. Whereas here we're not saying there is an acutal program - we will not "test" this file, it will not be used for marking, so it's more like main_example. Thoughts? I just lean that way but am not going to take it to war haha